policyengine_us/variables/gov/states/ca/ca_foster_care_minor_dependent.py

```diff
@@ -0,0 +1,16 @@
+from policyengine_us.model_api import *
+
+
+class ca_foster_care_minor_dependent(Variable):
+    value_type = bool
+    entity = Person
+    label = "California foster care minor dependent"
+    definition_period = YEAR
+    defined_for = StateCode.CA
+
+    def formula(person, period, parameters):
+        age = person("age", period) * MONTHS_IN_YEAR
```

Comment:
You sure this is needed? This is a yearly variable. Add a unit test.

I'd also suggest we just add the `monthly_age` variable in this PR.

Reply:
added unit test and converted to monthly